css/tutorial.styl

```diff
@@ -22,6 +22,8 @@
     text-align: center
 
   .media-card.tutorial-step
+    max-width: 400px
```

Comment:
I think I'm still seeing a little of the next step's image?
![screen shot 2018-05-10 at 3 50 53 pm](https://user-images.githubusercontent.com/12118751/39893621-f7430694-5469-11e8-821e-410def8e6618.png)

editing the `.tutorial-steps` `max-width` to `400px` seemed to help, but I'm not sure if that's a good approach.